gef-ui/server/utils/mocks/mock_applications.js

```diff
@@ -774,4 +889,5 @@ module.exports = {
   MOCK_DEAL,
```

Comment:
what is the difference between `MOCK_DEAL` and everything else?

Reply:
this is an old deal used by old tests.  have renamed to MOCK_BASIC_DEAL as it seems to be a basic deal for some tests to obtain values from